app/database/operator/server_data_operator/handlers/custom_profile.ts

```diff
@@ -47,7 +49,7 @@ const CustomProfileHandler = <TBase extends Constructor<ServerDataOperatorBase>>
             createOrUpdateRawValues,
             tableName: CUSTOM_PROFILE_FIELD,
             prepareRecordsOnly,
-        }, 'handleCustomProfileFields');
+        }, FIELD_HANDLER_DESCRIPTION);
```

Comment:
Any special reason to keep these in constants? These strings are only for debug purposes, so having them shared in different parts of the code could be even counter productive.

Reply:
No this is a personal tendency for reasons that doesn't apply here (but I just go that route almost always).  I'll switch back to have a string directly.